docs/UserGuide.adoc

```diff
@@ -237,6 +237,64 @@ Selects the 2nd person in the address book.
 `select 1` OR `s 1` +
 Selects the 1st person in the results of the `find` command.
 
+=== Adding an order : `orderadd` `since v1.2`
+
+Adds an order to the person specified by the index number used in the last person listing. +
```

Comment:
I understand that currently orders are added to list of orders instead of persons in V1.2, but perhaps for now description of this command can omit the details on adding orders to specified person?

Reply:
Yeah, but the thing is the command format used in the application remains the same. Not sure if I can write a different command format in the User Guide but use another one in the actual application?

Reply:
I thought I could update it in V1.3?

Reply:
So currently addorder format includes INDEX?

Reply:
Yeah

Reply:
In that case, do you want to put INDEX in the format in line 243?

Reply:
Oh thanks! I missed that. Will update

Reply:
Ah, I see. It looks ready to merge.